client/src/models/Contact.ts

```diff
@@ -1,9 +1,9 @@
 interface Contact {
-    serialId?: number;
+    serialId: number;
```

Comment:
Why did you remove the '?'

Reply:
because it is always -1 or the value from the DB so it is always defined

Reply:
ok

Reply:
So I think you can delete the condition in line 102 in InteractionEventForm.
I added this condition to solve a bug that cause saving multiple contacts every update - make sure it works property